cli/gobuster.go

```diff
@@ -126,7 +148,7 @@ func progressWorker(c context.Context, g *libgobuster.Gobuster, wg *sync.WaitGro
 }
 
 func writeToFile(f *os.File, output string) error {
-	_, err := f.WriteString(fmt.Sprintf("%s\n", output))
+	_, err := f.WriteString(output)
```

Comment:
This kills normal text output to a file because you are removing the new line character. You could pass an option to `writeToFile` if we should include the newline character or not

Reply:
Updated to include an optional flag to append a newline character.